### PR TITLE
Validate BRAWNDO

### PR DESCRIPTION
**Pull request by BrawndoSOL**
# Validate [$BRAWNDO](https://solscan.io/token/DUuqTfp6CxceXiuaqeHoSwH2NoYegPBav8NmcE1azHQU)

## Attestations:
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support:
- https://x.com/BrawndoSOL/status/1771235514923770095?s=20
- https://x.com/BrawndoSOL/status/1770997379421847609?s=20
- Coingecko/ CMC URL(s)
- https://www.geckoterminal.com/solana/pools/CLoXzvFKcVAB1mPoL3CGJYvezyRx4fu2YGtXYHLHfLk1?utm_source=coingecko&utm_medium=referral&utm_campaign=searchresults
- https://coinmarketcap.com/dexscan/solana/CLoXzvFKcVAB1mPoL3CGJYvezyRx4fu2YGtXYHLHfLk1/#About

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by BrawndoSOL — COMMENTED** [review]
Updated X in the brackets and deleted the 2nd X link

File: BRAWNDO [hunk 1]
```diff
@@ -3,18 +3,17 @@
 ## Attestations:
 - Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support:
 - https://x.com/BrawndoSOL/status/1771235514923770095?s=20
-- https://x.com/BrawndoSOL/status/1770997379421847609?s=20
```
Comment (BrawndoSOL) [inline BRAWNDO]: Deleted

File: BRAWNDO [hunk 2]
```diff
@@ -3,18 +3,17 @@
...
-- Coingecko/ CMC URL(s)
```
Comment (BrawndoSOL) [inline BRAWNDO]: Fixed

File: BRAWNDO [hunk 3]
```diff
@@ -3,18 +3,17 @@
...
+- Coingecko/ CMC URL
 - https://www.geckoterminal.com/solana/pools/CLoXzvFKcVAB1mPoL3CGJYvezyRx4fu2YGtXYHLHfLk1?utm_source=coingecko&utm_medium=referral&utm_campaign=searchresults
 - https://coinmarketcap.com/dexscan/solana/CLoXzvFKcVAB1mPoL3CGJYvezyRx4fu2YGtXYHLHfLk1/#About
 
 ## Validation (Please check off boxes):
-- [Y] The metadata provided in the PR matches what is on-chain (Mandatory)
```
Comment (BrawndoSOL) [inline BRAWNDO]: Fixed

File: BRAWNDO [hunk 4]
```diff
@@ -3,18 +3,17 @@
...
-- [Y] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
```
Comment (BrawndoSOL) [inline BRAWNDO]: fixed

File: BRAWNDO [hunk 5]
```diff
@@ -3,18 +3,17 @@
...
-- [Y] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)
```
Comment (BrawndoSOL) [inline BRAWNDO]: fixed

File: BRAWNDO [hunk 6]
```diff
@@ -3,18 +3,17 @@
...
 ## Validation (Please check off boxes):
-- [Y] The metadata provided in the PR matches what is on-chain (Mandatory)
-- [Y] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
-- [Y] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  
+- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
+- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
+- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  
 
 ## Acknowledgement (Please check off boxes)
-- [Y] My change matches the format in the file (no spaces between fields).
```
Comment (BrawndoSOL) [inline BRAWNDO]: fixed

File: BRAWNDO [hunk 7]
```diff
@@ -3,18 +3,17 @@
...
-- [Y] My token is already live and trading on Jupiter.
```
Comment (BrawndoSOL) [inline BRAWNDO]: fixed

File: BRAWNDO [hunk 8]
```diff
@@ -3,18 +3,17 @@
...
-- [Y] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
```
Comment (BrawndoSOL) [inline BRAWNDO]: fixed

File: BRAWNDO [hunk 9]
```diff
@@ -3,18 +3,17 @@
...
-- [Y] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.
```
Comment (BrawndoSOL) [inline BRAWNDO]: fixed

**Review by BrawndoSOL — COMMENTED** [review]
Updated errors

**Comment by BrawndoSOL** [comment]
What am I missing? Thank you.

**Review by BrawndoSOL — COMMENTED** [review]
Updated

**Comment by Jymfletzfur** [comment]
submission is incorrect, but token does not meet volume requirements

**Pull request closed without merging**